docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+Governance
+==========
+
+
+Overview
+--------
+
+Tedana is a relatively small open source project that requires specialized
+knowledge in multiple domains. This leads to several challenges. No one
+person on the current tedana development team has a combination of 
+available time plus expertise in collaborative software development, MRI
+physics, and advanced data processing methods to assume a primary project
+leader role. Even if such a person was interested, it may not benefit the
+project to overly rely on the existence of one person. We developed the
+following system the following goals in mind:
+
+- Promote open discussions.
+- Minimize the administrative burden.
+- Strive for consensus.
+- Provide a path for when consensus cannot be achieved.
+- Grow the community.
+- Maximize the `bus factor`_ of the project.
+- Acknowledge the leadership and time multiple people contribute to our
+  community without demanding more time than any individual can offer.
+  Dividing leadership responsabilies into multiple smaller roles also
+  makes it easier to encourage new people to take on a leadership role
+  without fearing that too much work will be required of them.
+
+This governance structure is a work-in-progress. We welcome both people
+who want to take on a leadership role as well as ideas for our to improve
+this structure.
+
+
+
+Steering Committee and Leadership Roles
+---------------------------------------
+
+Organizational Principles
+`````````````````````````
+The steering committee steers. The goal of the steering committee is to help
+guide the direction of the project. Decisions in the steering committee will 
+focus on how to present project issues to the broader community in a clear way
+rather than making project decisions without community input. 
+
+Issues or pull requests that can be reviewed by any community member can be
+reviewed and approved by the steering committee or steering committee members.
+The steering committee can decide that an issue should be prioritized for wider
+communal discussion or that a pull request requires more discussion or reviews
+than standard before merging. The steering committee can also decide how a
+breaking change (something that changes existing user function calls or program
+outputs) will be presented to the developer and user base for discussion, before
+decisions are made.
+
+Steering committee decisions should strive for consensus and accept the
+majority’s decision. If voting is necessary, it can happen asynchronously and is
```

Comment:
Which majority ? The following sentences makes it seem like this is an internal steering committee vote, rather than a community vote, but it's not completely clear to me.

Reply:
The idea is that the steering committee has the power to decide how we engage with the broader community and get feedback in situations where communal voices are necessary for the decision process. I agree we can be more specific, but I'll need to think about how to add specificity without **unnecessarily** reducing administrative flexibility